commit 91026083c2a4c9cc57ed1c90e474cda8a5b7843a
Author: Ke Wu <mikewu@google.com>
Date:   Tue Nov 8 15:34:59 2016 -0800

    execute startup scripts from /var/lib/google
    
    /tmp is not executable on GCI. So run startup scripts from
    /var/lib/google.

diff --git a/google_compute_engine/metadata_scripts/script_manager.py b/google_compute_engine/metadata_scripts/script_manager.py
index 2993c36..98ad820 100755
--- a/google_compute_engine/metadata_scripts/script_manager.py
+++ b/google_compute_engine/metadata_scripts/script_manager.py
@@ -38,7 +38,8 @@ def _CreateTempDir(prefix):
   Yields:
     string, the temporary directory created.
   """
-  temp_dir = tempfile.mkdtemp(prefix=prefix + '-')
+  temp_dir = tempfile.mkdtemp(prefix=prefix + '-',
+                              dir='/var/lib/google')
   try:
     yield temp_dir
   finally:
